packages/wdio-runner/src/index.js

```diff
@@ -382,6 +377,7 @@ export default class Runner extends EventEmitter {
         }
 
         await global.browser.deleteSession()
+        process.env.WDIO_SYNC_ENABLED = false
```

Comment:
Is it allowed to use boolean here? I assumed that only strings are allowed

Reply:
removed that part